prac_08/dynamic_labels.kv

```diff
@@ -0,0 +1,4 @@
+BoxLayout:
```

Comment:
This layout is so simple you almost think you've got something wrong right?  Lucky Lindsay says how simple it will be in the Prac documents.